### PR TITLE
Group some files to their own folders

### PR DESCRIPTION
**Pull request by RubenRBS**
This PR moves some files around to group them in their own folders so there are less files in the big folders and stuff is easier to find.
Hopefully I did not miss anything and this is an easy merge, but the webpack version hash might need to get bumped

In `etc/scripts/`, added `disasms/`, `docenizers/`, and `util/` folders
In `lib/`, added `mapfiles/`, and `parsers/` folders (also moved `google.js` to
`shortener`)
In `static/`, added `widgets/` folder

I also added the cypress folder to .gitignore as those are only the results of the tests (Except for the `integration` folder where the tests live)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -27,3 +27,10 @@ newrelic_agent.log
 
 # approval tests
 *.received.*
+
+# cypress
```
Comment (mattgodbolt) [inline .gitignore:31]: Maybe move this to a `.gitignore` inside the `cypress` directory to keep this specific stuff in one place (and make it easier if it moves again)

**Review by mattgodbolt — APPROVED** [review]
Thanks! Long overdue :)

**Comment by partouf** [comment]
better to merge this sooner rather than later

**Review by matsjla — APPROVED** [review]
Maybe we would want to separate frontend tests from backend tests in the future? Anyways I'm sure this is more than good for now!!

**Comment by partouf** [comment]
> Maybe we would want to separate frontend tests from backend tests in the future? Anyways I'm sure this is more than good for now!!

I think this is already the case, unless I'm missing something... Backend tests are in `/test`, frontend tests are in `/static/tests` and `/cypress/integration`

**Comment by RubenRBS** [comment]
🤞 hope nothing breaks

**Comment by mattgodbolt** [comment]
This looks great and is now live :)

**Pull request merged**